references.adoc

```diff
@@ -0,0 +1,230 @@
+ * https://cookiecutter.readthedocs.io/en/latest/readme.html[Cookiecutter]: A command-line utility for creating boilerplate project files from cookiecutters (project templates).
+  * `$ vimtutor`
+
+== Computer Science and Math
+=== Algorithms and Complexity
+ * https://imgur.com/gallery/voutF[Sorting Algorithms Visualized]
+ * https://www.toptal.com/developers/sorting-algorithms[Sorting Algorithms Animations]
+ * https://github.com/TheAlgorithms/Python[All algorithms implemented in Python (for education)]
+ * http://cooervo.github.io/Algorithms-DataStructures-BigONotation/index.html[big O cheat sheet]
+ * http://jeffe.cs.illinois.edu/teaching/algorithms/[A Free Advanced Comprehensive Algorithm Textbook]
+
+=== Data Structures
+
+=== Computer Architecture
+
+=== Discrete Math
+ * http://mfleck.cs.illinois.edu/building-blocks/index-sp2018.html[An Undergrad Level Introduction to Discrete Math]
+ 
+=== Linear Algebra
+ * https://github.com/fastai/numerical-linear-algebra[Computational Linear Algebra]
+ * http://arminstraub.com/teaching/linearalgebra-fall14[Introduction to Applied Linear Algebra]
+
+=== Statistics
+ * http://www-bcf.usc.edu/~gareth/ISL/[An Introduction to Statistical Learning (with Applications in R)]
+ * https://daviddalpiaz.github.io/stat400sp18/[Introduction to Discrete and Continious Probability and Statistics]
+
+=== Logic
+
+== Careers in Tech
+
+=== Software Engineering
+ * https://blog.alicegoldfuss.com/how-to-get-into-sre/[How to Get Into SRE]
+ * https://www.oodesign.com[Object oriented design patterns]
+ * https://github.com/fbeline/design-patterns-JS[23 Design Patterns Implemented in Javascript]
+ * https://github.com/faif/python-patterns[A collection of design patterns and idioms in Python]
+
+=== Data Science
+ * https://github.com/MrMimic/data-scientist-roadmap[Data Science Roadmap]
+ 
+=== Electrical Engineering
+
+=== Artificial Intelligence and Machine Learning
+ * https://www.deeplearningbook.org/[The Deep Learning Textbook]
+ * https://www.tensorflow.org/tutorials/[Get Started with TensorFlow]
+ * https://keras.io/[keras]
+
+=== Game Development
+
+=== Information Security
+ * https://www.hacksplaining.com/[Hacksplaining]
+
+=== Web Development
+ * https://tutorial.djangogirls.org/en/[Django Girls Tutorial]
+ * https://www.railstutorial.org/book[Ruby on Rails Tutorial by Michael Hartl]
+ * https://gist.github.com/jendiamond/5a26b531e8e47b4aa638[Rails Girls LA 2016]
+ * https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-i-hello-world[Flask Mega Tutorial by Miguel Grinberg]
+ * https://flask.palletsprojects.com/en/1.1.x/tutorial/[Offical Flask Tutorial from the Flask Documentation]
+
+=== Version Control
+ * https://ohshitgit.com/[Oh shit, git!]
+ * http://think-like-a-git.net/[Think Like (a) Git]
+
+=== Python
+ * https://realpython.com/python-virtual-environments-a-primer/[Python Virtual Environments: A Primer]
+ * https://automatetheboringstuff.com/[Automate the Boring Stuff with Python]
```

Comment:
I'm not sure about this one, but I could see an argument for tutorials, possibly how-to guides

Reply:
I think it probably fits best with how-to guides!